### PR TITLE
Lost points, Not show KC with no points

### PR DESCRIPTION
**Pull request by alexbojko**
* Adds section with lost points
* Fix to not to show KC if user has no points with suck KC

### CODE REVIEW

File: conversation/converters.py [hunk 1]
```diff
@@ -291,18 +296,18 @@ def convert_results_message(self, task, results_data):
...
         "effects": {
             "load": {
                 "effect": "none"
             },
-            "pullOutSegmentOnClick": {
-                "effect": "linear",
-                "speed": 400,
-                "size": 20
-            }
+            # "pullOutSegmentOnClick": {
```
Comment (rimikt) [inline conversation/converters.py]: Do we need this in future?
I prefer to remove unused code.

File: conversation/mongo.py [hunk 2]
```diff
@@ -143,28 +145,42 @@ def calculate_user_kc_scores(self, user_id, task_id):
...
             # history['KC'] contains a sum of all scores for that KC
             if kc_node_weights_sum[kc] != 0:
-                kc_scores[kc] = float(history.get('KC', {}).get(kc, 0)) / float(kc_node_weights_sum[kc])
+                kc_scores[kc] = {
+                    'user_owned': round(float(history.get('KC', {}).get(kc, 0)) / float(kc_node_weights_sum[kc]), 1),
+                    'max': round(float(kc_node_max_score[kc]) / float(kc_node_weights_sum[kc]), 1)
+                }
             else:
-                kc_scores[kc] = 0
+                kc_scores[kc] = {
```
Comment (rimikt) [inline conversation/mongo.py]: Add comment for this please.
Do we SHOW or NOT SHOW such KC's?

Comment (alexbojko) [reply conversation/mongo.py]: it will not show KC's with max score = 0

File: conversation/converters.py (hunk 1 above)
Comment (alexbojko) [reply conversation/converters.py]: I think we will need it in future.

**Pull request merged**